ui/dashboards/src/components/GridLayout/GridTitle.tsx

```diff
@@ -58,30 +56,28 @@ export function GridTitle(props: GridTitleProps) {
         padding: (theme) => theme.spacing(1),
         backgroundColor: (theme) => theme.palette.background.default,
       }}
-      onMouseEnter={() => setIsHovered(true)}
-      onMouseLeave={() => setIsHovered(false)}
     >
       {collapse ? (
         <>
           <IconButton onClick={collapse.onToggleOpen}>
             {collapse.isOpen ? <ExpandedIcon /> : <CollapsedIcon />}
           </IconButton>
           {text}
-          {isEditMode && isHovered && (
+          {isEditMode && (
             <Stack direction="row" sx={{ marginLeft: 'auto' }}>
-              <IconButton aria-label="add panel to group" onClick={openAddPanel}>
-                <AddIcon />
+              <IconButton aria-label={`add panel to group ${title}`} onClick={openAddPanel}>
+                <AddPanelIcon />
               </IconButton>
-              <IconButton aria-label="edit group" onClick={openEditPanelGroup}>
+              <IconButton aria-label={`edit group ${title}`} onClick={openEditPanelGroup}>
```

Comment:
Added the `title` so that (a) I could query for these buttons in the tests and (b) because seems to me that it doesn't hurt the accessibility to say the name of the panel